libr/config/config.c

```diff
@@ -593,21 +593,17 @@ R_API RConfigNode* r_config_set_i(RConfig *cfg, const char *name, const ut64 i)
 }
 
 R_API int r_config_eval(RConfig *cfg, const char *str) {
-	char *ptr, *a, *b, name[1024];
+	char *ptr, *config, *val, names[1024];
```

Comment:
why 1024, didnt tested the patch, but will be good to have some tests, does it support more than one assignment?

Reply:
Yes, it does suppport multiple assignments. 
i.e.
`e a = b = c = d = some_val`